README.md

```diff
@@ -47,9 +47,15 @@ A transaction is displayed.
 
 `npm start` and it will start running in the browser in http://localhost:3000
 
+### To Run in electron
+
+Before running using electron you have to execute `./node_modules/.bin/electron-builder install-app-deps`.
```

Comment:
Added a script on `package.json`. I have to update here.

Also, update ELECTRON.md with all commands.

Reply:
Done